packages/tabs/src/Tabs.tsx

```diff
@@ -4,41 +4,84 @@ import { css, cx } from '@leafygreen-ui/emotion';
 import { uiColors } from '@leafygreen-ui/palette';
 import { isComponentType, keyMap } from '@leafygreen-ui/lib';
 import { useEventListener } from '@leafygreen-ui/hooks';
+import { useUsingKeyboardContext } from '@leafygreen-ui/leafygreen-provider';
 import TabTitle from './TabTitle';
 import omit from 'lodash/omit';
 
-const borderHeight = 3;
+const Variant = {
```

Comment:
I've seen us define different enums for different components. In some cases we have `'default' | 'light`, but I've also seen `'dark' | 'light'`, `'default' | 'dark'`, and `'dark' | 'light'`. I imagine we'll want to have a standard convention if we'll eventually have some sort of provider that globally selects dark or light mode. just curious on what your thoughts are on this.

Reply:
So @DesignerDave's initial distinction between variants was if the colors are lighter than the default its considered `light`. I definitely agree with you in terms of our need to standardize this, and I think it could make sense to consider adding a `mode` prop and slowly deprecate variants when they only refer to `light`/`dark` in their discrepant ways. What do you think?